File: 1672.richest-customer-wealth.java
/*
 * @lc app=leetcode id=1672 lang=java
 *
 * [1672] Richest Customer Wealth
 */

// @lc code=start

import java.util.Arrays;

class Solution {
    public int maximumWealth(int[][] accounts) {
        int max = 0;
        for (int[] customer : accounts) {
            int wealth = Arrays.stream(customer).sum();
            if (wealth > max) {
                max = wealth;
            }
        }
        return max;
    }
}
// @lc code=end

File: 1672.richest-customer-wealth.cpp
/*
 * @lc app=leetcode id=1672 lang=cpp
 *
 * [1672] Richest Customer Wealth
 */

// @lc code=start

#include <vector>
#include <algorithm>
#include <numeric>

using namespace std;

class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        vector<int> amount;
        transform(accounts.begin(), accounts.end(), back_inserter(amount), [](const vector<int> &a) {
            return accumulate(a.begin(), a.end(), 0);
        });
        return *max_element(amount.begin(), amount.end());
    }
};
// @lc code=end
